tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/pages/404.jsx", "src/pages/404.jsx", "src/components/Error.jsx"],
```

Comment:
We should not replace the 404 page. It should exist when the URL searched is wrong. We should redirect users to the error page if they searched for `/events` or `/programs` or `/testimonials` only otherwise the page should be showing 404.